packages/odyssey-react-mui/src/TagList.tsx

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Stack, Tag } from ".";
+import { memo, ReactElement } from "react";
+
+export type TagListProps = {
+  children: ReactElement<typeof Tag> | Array<ReactElement<typeof Tag>>;
+};
+
+const TagList = ({ children }: TagListProps) => (
+  <Stack component="ul" direction="row" spacing={2}>
+    {children}
+  </Stack>
+);
```

Comment:
With MUI, we can't do this, but ideally, this would contain both the `ul` and `li` instead of it being separated out between two components. It's tight-coupling that I'd like to avoid if possible because it requires you know the source code of 2 components to use it.

I talk about it in this video: https://www.youtube.com/watch?v=n62Pc4KV4SM

Reply:
If we didn't have MUI constraints, what's the approach you would prefer? Make it a single component that accepts an array of tags, and renders a `<ul>` with the tag `<li>`s inside?